bodhi/server/services/updates.py

```diff
@@ -205,6 +205,11 @@ def set_request(request):
                            "Can't change request for an archived release")
         return
 
+    if update.status == UpdateStatus.stable and action == UpdateRequest.testing:
```

Comment:
Should we also look at the UI to make sure that the option to push to testing is not available when an update is stable ?

Reply:
mmm if I'm not mistaken this code should already prevent that:
https://github.com/fedora-infra/bodhi/blob/badf3695921e7f6969d2fcad453ce358b0df0c50/bodhi/server/templates/update.html#L31-L36
An update is 'pushed' since it entered testing, so the `push to testing` option shouldn't be available. In fact, I could only test that behavior by CLI, I've never seen that option in UI when an update is stable.

Reply:
Cool just wanted to make sure we covered all the cases :-)